ui/page/dcrdex/market.go

```diff
@@ -0,0 +1,1018 @@
+package dcrdex
+
+import (
+	"fmt"
+	"image/color"
+	"strconv"
+
+	"gioui.org/font"
+	"gioui.org/layout"
+	"gioui.org/unit"
+	"gioui.org/widget"
+
+	"github.com/crypto-power/cryptopower/app"
+	libutils "github.com/crypto-power/cryptopower/libwallet/utils"
+	"github.com/crypto-power/cryptopower/ui/cryptomaterial"
+	"github.com/crypto-power/cryptopower/ui/load"
+	"github.com/crypto-power/cryptopower/ui/modal"
+	"github.com/crypto-power/cryptopower/ui/page/components"
+	"github.com/crypto-power/cryptopower/ui/utils"
+	"github.com/crypto-power/cryptopower/ui/values"
+)
+
+const (
+	DEXMarketPageID = "dex_market"
+	// maxOrderDisplayedInOrderBook is the maximum number of orders that can be
+	// accommodated/displayed on the order book.
+	maxOrderDisplayedInOrderBook = 7
+)
+
+var (
+	u5                         = values.MarginPadding5
+	u8                         = values.MarginPadding8
+	u300                       = unit.Dp(300)
+	orderFormAndOrderBookWidth = (values.AppWidth / 2) - 40 // Minus 40 px to allow for margin between the order form and order book.
+	// orderFormAndOrderBookHeight is a constant height for the order form and
+	// orderbook. Use this to ensure they have the same height as they are
+	// displayed sided by side.
+	orderFormAndOrderBookHeight = unit.Dp(515)
```

Comment:
add this to dimensions.go file `values.MarginPadding515` just out of curiosity, is this what figma gave to you?

Reply:
> is this what figma gave to you?

 No, this is an arbitrary height that accommodates the current order form elements.